LoRaEngine/LoraKeysManagerFacade/FacadeStartup.cs

```diff
@@ -126,7 +191,18 @@ internal LocalConfigHandler()
 
                 internal override string RedisConnectionString => this.config.GetValue<string>(RedisConnectionStringKey);
 
-                internal override string IoTHubConnectionString => this.config.GetValue<string>(IoTHubConnectionStringKey);
+                internal override string IoTHubConnectionString => this.config.GetValue<string>(IoTHubConnectionStringKey, string.Empty);
```

Comment:
What's the reason for defaulting an empty string in these new settings? Seems it makes it unclear what's optional and what belongs together as a unit of settings.